.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+# The present workflow was made based on the following references:
+# - https://github.com/actions/cache/blob/main/examples.md#haskell---cabal
+# - https://github.com/haskell/time/blob/master/.github/workflows/ci.yml
+# - https://github.com/stackbuilders/stache/blob/master/.github/workflows/ci.yaml
+# - https://markkarpov.com/post/github-actions-for-haskell-ci.html
+---
+name: Build
+
+on: push
+
+concurrency:
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  haskell:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghc: ['8.2', '8.4', '8.6', '8.8', '8.10', '9.0']
```

Comment:
@cptrodolfox Thanks for all the good work!
May I ask again why `9.2` and `9.4` are not included in the CI?  GHC 9.2 is now the recommended version, and I would guess to also include testing the latest version.  (I didn't experience any build problems with these GHC versions.)

Reply:
Thanks, I have updated the build matrix to include 9.2 and 9.4.